src/core/models/alternativeverticals.js

```diff
@@ -13,15 +13,15 @@ export default class AlternativeVerticals {
 
   /**
    * Create alternative verticals from server data
-   * @param {Object} response The server response
-   * @param {Object.<string, function>} formatters The field formatters to use
+   * @param {Object[]} alternativeVerticals
    */
-  static from (response, formatters) {
-    const alternativeVerticals = response.alternativeVerticals;
-    if (!alternativeVerticals || !alternativeVerticals.modules) {
+  static from (alternativeVerticals) {
+    if (!alternativeVerticals || alternativeVerticals.length < 0) {
```

Comment:
Should this be `!alternativeVerticals || alternativeVerticals.length === 0`?

Reply:
Oh, yes 🙈